cardano-testnet/src/Testnet/Start/Types.hs

```diff
@@ -59,7 +59,7 @@ cardanoDefaultTestnetOptions = CardanoTestnetOptions
   , cardanoSlotLength = 0.1
   , cardanoTestnetMagic = 42
   , cardanoActiveSlotsCoeff = 0.05
-  , cardanoMaxSupply = 100_020_000_000 -- 100 billions. This amount should be bigger than the 'byronTotalBalance' in Testnet.Start.Byron
+  , cardanoMaxSupply = 100_000_020_000_000 -- 100 000 billions Lovelace, so 100 millions ADA. This amount should be bigger than the 'byronTotalBalance' in Testnet.Start.Byron
```

Comment:
This default (increased) amount was suggested by @CarlosLopezDeLara, who found the previous amount too low.

Reply:
Hmmm. We need to figure out a way to eliminate this discrepancy, preferably at the type level. Perhaps we can have a type called `CommonGenesisConfiguration`.

Reply:
@Jimbo4350> I'm revisiting this type anyway for the work on per-node-configuration, so we'll be able to improve more soon.

That is why I'm leaving it like this now.